server/updates_test.go

```diff
@@ -219,4 +230,14 @@ func TestUpdate(t *testing.T) {
 	lastLine = statusRes.Payload.LogLines[len(statusRes.Payload.LogLines)-1]
 	t.Logf("lastLine = %q", lastLine)
 	assert.Contains(t, lastLine, "PMM Server update finished")
+
+	// check that both pmm-managed and pmm-update were updated
+	version, err = serverClient.Default.Server.Version(nil)
+	require.NoError(t, err)
+	require.NotNil(t, version.Payload)
+	t.Logf("After update: %s", spew.Sdump(version.Payload))
+	assert.True(t, strings.HasPrefix(version.Payload.Managed.Version, version.Payload.Version),
+		"managed.version = %q should have %q prefix", version.Payload.Managed.Version, version.Payload.Version)
+	assert.True(t, strings.HasPrefix(version.Payload.Server.Version, version.Payload.Version),
+		"server.version = %q should have %q prefix", version.Payload.Server.Version, version.Payload.Version)
 }
```

Comment:
let's also check that the version is changed.

Reply:
It is not changed if we (force) update to the same version.